encoding/thrift/thriftrw-plugin-yarpc/main.go

```diff
@@ -64,7 +64,7 @@ package <$pkgname>
 // Interface is the server-side interface for the <.Service.Name> service.
 type Interface interface {
 	<if .Parent>
-		<$parentPath := printf "%s/yarpc/%sserver" .ParentModule.ImportPath (lower .Parent.Name)>
+		<$parentPath := printf "%s/%sserver" .ParentModule.ImportPath (lower .Parent.Name)>
```

Comment:
Does this stutter bother you much? (e.g., `echo/echoserver/server.go`) 
Can de do simple `%s/server`?